### PR TITLE
Implement patient transforms

### PR DESCRIPTION
**Pull request by graham-skylight**
* Name, phone, and geocoding

### CODE REVIEW

File: src/FunctionApps/python/IntakePipeline/transform.py [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+    """Grab a patient resource out of the bundle, and return a reference"""
+
+    patients = [
+        r
+        for r in bundle.get("entry")
+        if r.get("resource").get("resourceType") == "Patient"
+    ]
+
+    if patients:
+        return patients[0]
```
Comment (DanPaseltiner) [inline src/FunctionApps/python/IntakePipeline/transform.py:17]: This won't work if there are multiple patient resources in a bundle. In theory I don't think that should happen, but it is something to be aware of.

File: src/FunctionApps/python/IntakePipeline/transform.py [hunk 2]
```diff
@@ -1,2 +1,68 @@
...
+    if not resource:
+        return None
+
+    patient = resource.get("resource")
+
+    # Transform names
+    for name in patient.get("name", []):
+        if "family" in name:
+            name["family"] = transform_name(name["family"])
+
```
Comment (DanPaseltiner) [inline src/FunctionApps/python/IntakePipeline/transform.py:33]: Is family name only ever a single value and never a list?

**Review by DanPaseltiner — APPROVED** [review]

File: src/FunctionApps/python/IntakePipeline/transform.py (hunk 1 above)
Comment (bamader) [reply src/FunctionApps/python/IntakePipeline/transform.py:17]: I would prefer to see us code things to handle all patients in a bundle. While I do agree that theoretically that shouldn't happen, it's a small defensive thing we can do to make our code more robust and handle something I don't think we can/should safely assume.

File: src/FunctionApps/python/IntakePipeline/transform.py (hunk 2 above)
Comment (bamader) [reply src/FunctionApps/python/IntakePipeline/transform.py:33]: I've only ever seen it as a single value. Given name comes as a list, but I haven't seen that for family name. It's something we could do defensively if we wanted but I'm fine doing single value for now.

**Review by bamader — CHANGES_REQUESTED** [review]
Other than accounting for multiple patients in a bundle, this all looks good to me.

File: src/FunctionApps/python/IntakePipeline/transform.py (hunk 2 above)
Comment (graham-skylight) [reply src/FunctionApps/python/IntakePipeline/transform.py:33]: Let's wait until we see it as a list somewhere, no need to get super defensive for the prototype

**Pull request merged**